tethys_gizmos/static/tethys_gizmos/js/cesium_map_view.js

```diff
@@ -272,23 +273,24 @@ var CESIUM_MAP_VIEW = (function() {
             return;
         }
 
-        let m_model_option_properties = cesium_options(m_models_options);
-        for (let m_model_option_property in m_model_option_properties)
-        {
-            var name = m_model_option_properties[m_model_option_property]['name'];
+        for (let m_model of m_models) {
+            // options is where the model data is store.
+            let m_model_option = cesium_options(m_model.options);
+            let m_model_data = m_model.data
```

Comment:
Variables declared in a local scope should not start with `m_`. This means "member" variable or global variable and is reserved for variables declared at the top. Use `model`, `model_options`, and `model_data` instead.